job-descriptions/forward-deployed-engineer.md

```diff
@@ -4,18 +4,15 @@
 
 ### About us
 
-At Sourcegraph, we are building a better, smarter foundation for software development. The innovations of the future will all rely on code and by empowering software developers today, we can bring the future sooner. You can learn more about our mission by reading our [our master plan](https://sourcegraph.com/plan).
+Sourcegraph is on a mission to make it so everyone, in every community, in every country, and in every industry can create products using the best technology. The innovations of the future will all rely on software, so by empowering software developers today we can bring the future sooner. You can learn more about our mission by reading our [our master plan](https://sourcegraph.com/plan).
 
-Our product (code search, code intelligence, browser extensions, etc.) is [open
-source](https://about.sourcegraph.com/blog/sourcegraph-is-now-open-source), and is already deployed
-to paying customers with small and large engineering organizations across the world. Visit [our
-homepage](https://sourcegraph.com/start) to learn why companies use Sourcegraph.
+We are an [open company](https://docs.sourcegraph.com/dev/open_source_open_company): our code, our product roadmap, and our company processes are public because this transparency helps us rapidly gather feedback, iterate, learn, and deliver the best product to our customers.
```

Comment:
```suggestion
We are an [open company](https://docs.sourcegraph.com/dev/open_source_open_company). Our code, our product roadmap, and our company processes are public. This transparency helps us rapidly gather feedback, iterate, learn, and deliver the best product to our customers.
```

Reply:
I'll remove the "because this" and use two sentences (this is actually what I had initially and waffled back and forth. Given that change, are you ok if I leave the colon? I feel like it flows a bit better than separate sentences for the first part.

```suggestion
We are an [open company](https://docs.sourcegraph.com/dev/open_source_open_company): our code, our product roadmap, and our company processes are public. This transparency helps us rapidly gather feedback, iterate, learn, and deliver the best product to our customers.
```